carbonmark/components/Carousel/Carousel.tsx

```diff
@@ -0,0 +1,149 @@
+import { cx } from "@emotion/css";
+import InfoOutlinedIcon from "@mui/icons-material/InfoOutlined";
+import KeyboardArrowLeftOutlinedIcon from "@mui/icons-material/KeyboardArrowLeftOutlined";
+import KeyboardArrowRightOutlinedIcon from "@mui/icons-material/KeyboardArrowRightOutlined";
+import { ProjectMap } from "components/pages/Project/ProjectMap";
+import { TextInfoTooltip } from "components/TextInfoTooltip";
+import useEmblaCarousel, { EmblaOptionsType } from "embla-carousel-react";
+import { useResponsive } from "hooks/useResponsive";
+import { CarouselImage, Project } from "lib/types/carbonmark";
+import { useCallback, useEffect, useState } from "react";
+import * as styles from "./styles";
+
+type CarouselProps = {
+  images: Array<CarouselImage>;
+  options?: EmblaOptionsType;
+  location?: Project["location"];
+};
+
+const Carousel: React.FC<CarouselProps> = (props) => {
+  const { isMobile } = useResponsive();
+  const [maxSlidesToShow] = useState(5);
+  const [selectedIndex, setSelectedIndex] = useState(0);
+  const [images, setImages] = useState<Array<CarouselImage>>(props.images);
+  const [mainRef, mainApi] = useEmblaCarousel(props.options);
+  const [thumbsRef, thumbsApi] = useEmblaCarousel({
+    containScroll: "keepSnaps",
+    dragFree: true,
+  });
+
+  const onThumbClick = useCallback(
```

Comment:
nitpick, lets go with the onClick -> handleClick convention I outlined here:
https://github.com/KlimaDAO/klimadao/wiki#use-an-oneventhandleevent-naming-convention